crates/node_binding/src/lib.rs

```diff
@@ -38,16 +38,22 @@ impl Rspack {
     env: Env,
     options: RawOptions,
     builtin_plugins: Vec<BuiltinPlugin>,
+    parser_plugins: Vec<JsParserPlugin>,
```

Comment:
I think there could have a way to just use BuiltinPlugin here instead of introducing a new JsParserPlugin

Reply:
Any advice to implement this?